src/test/java/de/metas/ui/web/comments/json/JSONCommentCreateRequestTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * #%L
+ * metasfresh-webui-api
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.ui.web.comments.json;
+
+import de.metas.util.JSONObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JSONCommentCreateRequestTest
+{
+	@Test
+	void testSerialisationDeserialisation()
+	{
+		final JSONObjectMapper<JSONCommentCreateRequest> jsonObjectMapper = JSONObjectMapper.forClass(JSONCommentCreateRequest.class);
+
+		final JSONCommentCreateRequest expected = new JSONCommentCreateRequest("This is a test Comment.\nTra la la.");
+
+		final String json = jsonObjectMapper.writeValueAsString(expected);
+		final JSONCommentCreateRequest deserialisedRequest = jsonObjectMapper.readValue(json);
+		assertThat(deserialisedRequest).isEqualToIgnoringGivenFields(expected);
+		System.out.println(deserialisedRequest);
```

Comment:
instead of cluttering the build output, consider storing a snapshot. that way we keep an automated check on all field's values (you can use SystemTime to control timestamps)

Reply:
i simply forgot to remove the sout.